rclcpp/include/rclcpp/create_timer.hpp

```diff
@@ -60,8 +60,8 @@ create_timer(
   rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr)
 {
   return create_timer(
-    rclcpp::node_interfaces::get_node_base_interface(node),
-    rclcpp::node_interfaces::get_node_timers_interface(node),
+    rclcpp::node_interfaces::get_node_base_interface(node).get(),
+    rclcpp::node_interfaces::get_node_timers_interface(node).get(),
```

Comment:
wouldn't it make sense to change the `create_timer` API to take a shared pointer? Seems kind of odd that other `create_x` functions take shared pointers, but timers don't, e.g. https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/create_client.hpp#L33-L35

Reply:
That's true, but I didn't want this PR to cover more ground than it needs to. We can correct that in a follow-up PR.